packages/soba/controls/src/lib/orbit-controls/orbit-controls.directive.ts

```diff
@@ -132,12 +125,21 @@ export class NgtSobaOrbitControls
 
   #setControls = this.effect<THREE.Camera | undefined>((params$) =>
     params$.pipe(
-      withLatestFrom(this.selectors.enableDamping$),
-      tap(([camera, enableDamping]) => {
+      withLatestFrom(
+        this.selectors.enableDamping$,
+        this.selectors.target$
+      ),
+      tap(([camera, enableDamping, target]) => {
         this.ngZone.runOutsideAngular(() => {
           if (camera) {
             const controls = new OrbitControls(camera);
             controls.enableDamping = enableDamping;
+            if (target) {
+              const controlsTarget: Vector3 | undefined = makeVector3(target);
+              if (controlsTarget) {
+                controls.target = controlsTarget;
+              }
+            }
```

Comment:
suggestion: since `target` is already null-checked, `makeVector3` should return a value. So either not checking target then we'll check `controlsTarget` or we don't have to check `controlsTarget`

```ts
if (target) {
   controls.target = makeVector3(target)!;
}

// or
const controlsTarget = makeVector3(target);
if (controlsTarget) {
   controls.target = controlsTarget;
}
```

Reply:
Ok, I didn't notice `target` was already null checked.

Reply:
Can you make the changes?